patient_portal/messaging/messages.php

```diff
@@ -679,4 +679,4 @@ class="pull-right glyphicon glyphicon-warning-sign text-danger"></span></span> -
     </ng>
 
 </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
Thank you for adding a new line.

Reply:
I'm using Atom, and it did it automatically :D
Will look at it tomorrow..